docs/modules/ROOT/pages/how-to-guides/proc_prefetching-dependencies-to-support-hermetic-build.adoc

```diff
@@ -201,6 +203,39 @@ spec:
 
 include::../partials/con_hermetic_verification.adoc[]
 
+== Enabling prefetch builds for `npm`
+Cachi2 supports `npm` by fetching any dependencies you declare in your `package.json` and `package-lock.json` project files. The npm CLI manages the `package-lock.json` file automatically, and Cachi2 downloads the fetched dependencies to your `deps/npm/` output directory as ``.tar` files.
+
+.Prerequisites
+* You have an link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/configuring-builds/proc_upgrade_build_pipeline/[upgraded build pipeline].
+* You have an up-to-date link:https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json[`package-lock.json`] file in your source repository. To make sure that you have the latest `package-lock.json` file, or to create a lockfile, run the link:https://docs.npmjs.com/cli/v9/commands/npm-install?v=true[`npm-install`] command.
+* You have a `package.json` file that's newer than version 1.
```

Comment:
The version constraint is on the `package-lock.json` file, not the `package.json` file. It may also be good to mention how to check whether the package.lock.json file has a supported version - i.e. check that the [`lockfileVersion`](https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json#lockfileversion) attribute says `2` or `3`